core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2817,6 +2817,22 @@ struct ViewDataHandle<
 namespace Kokkos {
 namespace Impl {
 
+template <typename T>
```

Comment:
Why was this moved?

Reply:
Moving it here compiles, i.e. the include order works this way, leaving it in the original place does not.
We could of course define it in a separate header file instead.

Reply:
This is fine by me, just curious